image_writers_guide/sti_testing.adoc

```diff
@@ -77,37 +46,50 @@ build:
 test:
 	docker build -t $(IMAGE_NAME)-candidate .
 	IMAGE_NAME=$(IMAGE_NAME)-candidate test/run
----
-
-== Basic work flow
-
-This section describes the default work flow you should be using for testing the
-STI image builders:
-
-First build the STI image using 'docker build -t BUILDER_IMAGE_NAME'
-
-1. docker run BUILDER_IMAGE_NAME <verify the usage script is working>
-1. sti build file:///path-to-sample-app BUILDER_IMAGE_NAME OUTPUT_APPLICATION_IMAGE_NAME
-1. <in case you support save-artifacts, execute 2. step again to verify that
-   restoring artifacts works properly>
-1. docker run OUTPUT_APPLICATION_IMAGE_NAME
-1. <verify the container is running and the application is responding>
-
-Executing all the steps above, should be usually enough to tell if the STI
-commands operates properly.
+----
+====
+
+== Basic Testing Workflow
+The `test` script assumes that there is an image to test already built. If required, first build the STI image using:
```

Comment:
[Suggestion for readability]
The `test` script assumes you have already built the image that you want to use for testing.